app/questionnaire/path_finder.py

```diff
@@ -34,7 +34,7 @@ def routing_path(
         Visits all the blocks in a section and returns a path given a list of answers.
         """
         blocks: List[Mapping] = []
-        routing_path_block_ids = []
+        routing_path_block_ids: List = []
```

Comment:
Collections should define what type they hold.

Reply:
Added type to Collection, thanks!